blockly-core/build-output/blockly_uncompressed.js

```diff
@@ -23437,7 +23437,7 @@ Blockly.BlockSpaceEditor.prototype.createDom_ = function(container) {
   svg.appendChild(this.blockSpace.createDom());
```

Comment:
The staging server builds them and checks in the updated versions.  Then test & prod serve the built versions.  No need to check in changes since they'll be overwritten by staging build (and can cause merge conflicts in rare situations).